subjects/devops/play-with-containers-py/audit/README.md

```diff
@@ -0,0 +1,225 @@
+- `billing-db container` is a SQL database server that contains your billing database, it must be accessible via port `5432`.
+- `inventory-app container` is a Node.js server that contains your inventory-app. It will be connected to the inventory database and accessible via port `8080`.
+- `billing-app container` is a Node.js server that contains your billing-app. It will be connected to the billing database and consuming the messages from the RabbitMQ queue. It will be accessible via port `8080`.
+- `rabbit-queue` is a RabbitMQ server that contains the queue.
+- `api-gateway-app container` is a Node.js server that contains your api-gateway-app. It will forward the requests to the other services and it's accessible via port `3000`.
+
+##### Check the Container restart policy:
+
+```console
+user:~$ docker inspect -f "{{ .HostConfig.RestartPolicy }}" <container-name>
+{on-failure 0}
+user:~$
+```
+
+###### Do all containers have the correct configuration?
+
+###### Are the containers configured to restart in case of failure?
+
+#### Check the Docker volumes:
+
+```console
+user:~$ docker volume ls
+DRIVER    VOLUME NAME
+<...>     inventory-database
+<...>     billing-database
+<...>     api-gateway-app
+user:~$
+```
+
+- `inventory-database volume` contains the inventory database.
+- `billing-database volume` contains the billing database.
+- `api-gateway-app volume` contains the API gateway logs.
+
+###### Do all volumes have the correct configuration?
+
+##### Ask the following questions to the group or student
+
+##### What is a Docker volume?
+
+##### Why do we use Docker volumes?
+
+###### Did the student reply correctly to the questions?
+
+#### Check the solution network:
+
+###### Is the connection to the api-gateway-app the only one exposed from outside of the Docker host?
+
+##### Ask the following questions to the group or student
+
+##### What is the Docker network?
+
+##### Why do we use the Docker network?
+
+###### Did the student reply correctly to the questions?
+
+#### Check the Docker images:
+
+```console
+user:~$ docker images
+REPOSITORY              TAG          IMAGE ID       CREATED        SIZE
+inventory-database     <...>          <...>         <...>          <...>
+billing-database       <...>          <...>         <...>          <...>
+inventory-app          <...>          <...>         <...>          <...>
+billing-app            <...>          <...>         <...>          <...>
+RabbitMQ               <...>          <...>         <...>          <...>
+api-gateway-app        <...>          <...>         <...>          <...>
+user:~$
+```
+
+###### Is there a Docker image for each service with the same service name?
+
+##### Ask the following questions to the group or student
+
+##### What is a Docker image?
+
+##### Why do we use Docker images?
+
+##### Where can we find some public Docker images?
```

Comment:
this question is not explicit in the readme, the student wont know that this is a "must know"

Reply:
I see what you mean :) I will remove the question to be sure people won't think it is unfair